devices/patches/usb_otg/fs_l4+_global.yaml

```
@@ -0,0 +1,272 @@
+GOTGCTL:
+  _add:
+    VBVALOEN:
+      description: VBUS valid override enable
+      bitOffset: 2
+      bitWidth: 1
+    VBVALOVA:
```

From what I can tell, RM0432 calls this
```suggestion
    VBVALOVAL:
```